less/home-page.less

```
@@ -1,75 +1,115 @@
 // Home Content
-
 .home {
-
-  .intro {
-
-    h2 {
-      padding: 15px;
+    .intro {
+        display: flex;
+        h2 {
+            padding: 15px;
+            align-self: flex-start;
+        }
+        p {
+            padding: 0 15px;
+        }
+        @media @mobile {
+            img {
+                width: 100%;
+            }
+        }
     }
-
-    p {
-      padding:0 15px;
-    }
-  
-  }
-
-  .space-images {
-    display: flex;
-    flex-wrap: wrap;
-    width: 100%;
-
-    img {
-      width: 28%;
-      margin: 20px 2.5%;
-      border-radius: 20px;
-      height: 100%;
-    }
-  }
-
-  .visit-planets {
-    padding: 15px;
-
-    h3 {
-      margin-top: 11px;
+    .space-images {
+        display: flex;
+        flex-wrap: wrap;
+        width: 100%;
+        @media @mobile {
+            .sky,
+            .launch {
+                display: none;
+            }
+            .blast {
+                width: 100%;
+            }
+        }
+        img {
+            width: 28%;
+            margin: 20px 2.5%;
+            border-radius: 20px;
+            height: 100%;
+        }
     }
-
-    p {
-      margin-top: 13px;
+    .visit-planets {
+        padding: 15px;
+        h3 {
+            margin-top: 11px;
+        }
+        p {
+            margin-top: 13px;
+        }
     }
-  }
-
-  .walk {
-    margin-top: 60px;
-    border-top: 1px solid @main-border-color;
-    padding: 50px 60px;
-    display: flex;
-    justify-content: center;
-
-    .walk-text {
-      width: 50%;
-
-      h3 {
-        margin-bottom: 10px;
-      }
-
-      p {
-        margin-bottom: 26px;
-      }
-
-      .btn {
-      }
+    .walk {
+        margin-top: 60px;
+        border-top: 1px solid @main-border-color;
+        padding: 50px 60px;
+        display: flex;
+        justify-content: center;
+        .walk-text {
+            width: 50%;
+            h3 {
+                margin-bottom: 10px;
+            }
+            p {
+                margin-bottom: 26px;
+            }
+            .btn {
+                .btn-style(70px, 30px, 1.7rem, 6px);
+                a { 
+                  text-decoration: none; 
+                  color: @sea-shell;
+                }
```

```suggestion
     .btn {
         a { 
             .btn-style(70px, 30px, 1.7rem, 16px);
             text-decoration: none; 
             color: @sea-shell;
            }
```
Move this mixin to the actual a tag, and give it a bit more padding. You'll have to play around with the actual values. See my other suggestions below about how to change the padding parameter for this mixin to make this a bit more buttony.